docs/deploy-cephfs.md

```diff
@@ -164,7 +164,7 @@ for more information.
 **Deploy Ceph configuration ConfigMap for CSI pods:**
 
 ```bash
-kubectl create -f ../example/ceph-config.yaml
+kubectl create -f ../../../example/ceph-config.yaml
```

Comment:
Can you explain in the commit message why this is needed? If the file is relative to this document, a single `../` seems sufficient.

Note that other files are referenced without a path. If this gets corrected, other references should be adapted too.

Reply:
I know is not explicit, but all the previous steps kind of assume that you are on `deploy/cephfs/kubernetes` folder, making the current relative path to the `example` folder not work.

If this reason is good enough for you, I can update the commit message.

Reply:
Sure, that's fine with me. It would be great if you can update the commit message and adjust the subject of your commit as well. I recommend `doc: update relative path to ceph-config.yaml file`, the `doc:` prefix should be valid for the **commitlint** check. Make sure to add your `Signed-off-by` as well.

Reply:
Thank you for the pointers. I hadn't noticed the CI failures yet.